### ISSUE TITLE
Combinator equivalence issue

### ISSUE THREAD

**Issue by IsaacDynamo** [codereport/cl-cpp#4]
Hello Conor,

Recently I found your podcast's and YouTube channel, and I want to thank you for all the great content you created.

With one of your vods in the background I also wrote some code to play around with combinators.
However I'm not able to reproduce all of your results, and I think there is an issue in your combinator equivalence check.

Let me give you a contradicting example from the results of your own run.

I: ["KK", "WK", "SKS", **"Wκ"**, "Cκ", "SKC", "κC", "WCK", "WKK", "SKK"]
K: ["κW", "Cκ", "κS", "κC", "κB", **"Wκ"**, "κκ", "κΦ", "BκSC", "WSκB"]

According to your program `Wκ` is both equivalent to `I` and  `K`. Implying `I` is equivalent to `K` which is not true.

In https://www.youtube.com/live/PMsJl8fxcOU?feature=share&t=508 you also noticed that something strange is happening, but never followed up on it.

To recap your own conclusion, the equivalence check only looks at the second half of the function abstraction, so after the dot. And then `I` (λa.a) and `K` (λab.a) both have an '.a' and are incorrectly considered equivalent. 

This means that some of your found spellings on https://combinatorylogic.com/table.html are incorrect. For example `CSK` is not an λab.abb but an λab**c**.abb.

YouTube user @roiproutii also noticed this and placed a similar remark on the video.

Looking forward to the fourth video in the series where you fix the issue 😉
